fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java

```diff
@@ -552,7 +566,10 @@ private long getTableRowCount(Table table, Operator node) {
         if (table.isNativeTable()) {
             OlapTable olapTable = (OlapTable) table;
             List<Partition> selectedPartitions;
-            if (node.isLogical()) {
+            if (node.getOpType() == OperatorType.LOGICAL_BINLOG_SCAN ||
+                    node.getOpType() == OperatorType.PHYSICAL_STREAM_SCAN) {
+                selectedPartitions = new ArrayList<>(olapTable.getAllPartitions());
```

Comment:
Well, row count for `BinlogScan` is almost nonsense. Since the table row count would continuously grow and the initialize value has not effect for the plan but could only display a rough statistic

I have changed it to a constant `1`.